src/app/globals.css

```
@@ -24,3 +24,56 @@ video {
+
+@layer utilities {
+  /* dvh 안전 유틸: 지원되면 dvh, 아니면 svh → vh 폴백 */
+  .min-h-screen-safe {
+    min-height: 100vh;
+    min-height: 100svh;
+  }
+
+  @supports (height: 100dvh) {
+    .min-h-screen-safe {
+      min-height: 100dvh;
+    }
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

min-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)